src/core/server/rendering/bootstrap/bootstrap.js.template.hbs

```diff
@@ -78,12 +107,6 @@ if (window.__kbnStrictCsp__ && window.__kbnCspNotEnforced__) {
       {{/each}}
     ], function () {
       __kbnBundles__.get('entry/core/public').__kbnBootstrap__();
-
-      load([
-        {{#each styleSheetPaths}}
-          '{{this}}',
-        {{/each}}
-      ]);
```

Comment:
Stylesheets are now directly included in the document generated by `RenderingServiceSetup.render` instead.